force-app/main/default/chat/lwc/chat/chat.js

```diff
@@ -0,0 +1,85 @@
+//todo:: const labels in html
+import {LightningElement, track, wire} from 'lwc';
+import {createRecord, getRecord} from "lightning/uiRecordApi";
+import {subscribe, unsubscribe} from 'lightning/empApi';
+
+import notifyAboutMessageApex from '@salesforce/apex/ChatController.notifyAboutMessage';
+
+import MESSAGE_OBJECT from "@salesforce/schema/Message__c";
+import BODY_FIELD from "@salesforce/schema/Message__c.Body__c";
+
+import MESSAGE_ID_FIELD from "@salesforce/schema/Chat_Event__e.Message_Id__c";
```

Comment:
The names of the imports are a bit misleading.
MESSAGE_OBJECT
BODY_FIELD -> MESSAGE_BODY_FIELD
MESSAGE_ID_FIELD -> EVENT_MESSAGE_ID_FIELD